package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Ignite'Em",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/ignite-em/ig-iwacu.git",
+  "author": "Luc Abayo <luc.bayo@gmail.com>",
```

Comment:
Great work @abayo-luc, won't it be better to put `Ignite'em` as an author? What do you think?